### PR TITLE
Discover and fix new typos

### PR DESCRIPTION
**Pull request by szepeviktor**
- upgrade `typos`
- fix variable names

### CODE REVIEW

File: .github/workflows/spell-check.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Install typos
-        uses: taiki-e/install-action@cd5df4de2e75f3b819ba55f780f7bb8cd4a05a41 # v2.32.2
+        uses: taiki-e/install-action@fe9759bf4432218c779595708e80a1aadc85cedc # v2.46.10
```
Comment (xbasel) [inline .github/workflows/spell-check.yml:29]: What's the justification for the new version?

Comment (szepeviktor) [reply .github/workflows/spell-check.yml:29]: It discovers more typos. See the second commit please.

**Review by madolson — APPROVED** [review]
Nice! Thanks for this.

**Comment by szepeviktor** [comment]
Glad to contribute to Valkey.

**Pull request merged**